resources/js/Pages/SurveyResults/TerminOptions.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="flex flex-col">
+
+        <label class="inline-flex items-center mt-3">
+            {{ terminDate.getDate() + '.' + (terminDate.getMonth() + 1) + '.' + terminDate.getFullYear() + '   ' + terminDate.getHours() + ':' + terminDate.getMinutes() + '  Dauer: ' + option.duration + ' minuten' }}
+        </label>
```

Comment:
it would be Cleaner to Outsource this in a Component i think, but its fine you can leave it at this